doggystyle.css

```diff
@@ -0,0 +1,186 @@
+/*adoptadog stylesheet*/
+* {
+    box-sizing: border-box;
+}
+
+
+header {
+  width: 100%
+  border: none;
+  margin: none;
+  background-color: DarkCyan;
+  padding: 2%;
+}
+
+header {
+  display: flex;
+  flex-flow: row;
+}
+
+main {
+  margin-top: 0;
+  margin-right: 10%;
+  margin- bottom: 5%;
+  margin-left: 10%;
+}
+
+h3 {
+  font-size: 20px;
+  font-family: Gill sans, sans-serif;
+  color: chartreuse;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.checkouticon {
+  margin: 2%;
+}
+
+.checkbutton {
+  background-color: chartreuse;
+  color: white;
+  padding: 5px 30px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 70%;
+}
+
+nav {
+  width: 100%;
+  margin: 0 0 1%;
+  padding: .5%;
+  background-color: pink;
+}
+
+li {
+  display: inline;
+  padding: 1%;
+  font-size: 82%;
```

Comment:
yeah, use rems/ems for all fonts. If you set your base font the math is easy. You could do 0.82rem (but I'd argue to make it something that's based on dividing by 16) example....14/16 is 0.875 which is close and will make your font be 14px if you set your body font size to 16px.

https://developer.mozilla.org/en-US/docs/Web/CSS/font-size

I'd use rems in your project, read up on how to do the math and how they are used. It makes your life so much easier